setup.cfg

```diff
@@ -9,4 +9,4 @@ per-file-ignores =
     praw/models/reddit/mixins/__init__.py:F401
 
 [wheel]
```

Comment:
I think you can completely delete this section of the file now.

Reply:
Understood.